mailpoet/lib/AdminPages/Pages/EmailEditor.php

```diff
@@ -64,6 +85,26 @@ public function render() {
       ]
     );
 
+    // Renders additional script data that some components require e.g. PremiumModal. This is done here instead of using
+    // PageRenderer since that introduces other dependencies we want to avoid. Used by getUpgradeInfo.
+    $installer = new Installer(Installer::PREMIUM_PLUGIN_SLUG);
+    $inline_script_data = [
+      'mailpoet_premium_plugin_installed' => Installer::isPluginInstalled(Installer::PREMIUM_PLUGIN_SLUG),
+      'mailpoet_premium_plugin_active' => $this->servicesChecker->isPremiumPluginActive(),
+      'mailpoet_premium_plugin_download_url' => $this->subscribersFeature->hasValidPremiumKey() ? $installer->generatePluginDownloadUrl() : null,
+      'mailpoet_premium_plugin_activation_url' => $installer->generatePluginActivationUrl(Installer::PREMIUM_PLUGIN_PATH),
+      'mailpoet_has_valid_api_key' => $this->subscribersFeature->hasValidApiKey(),
+      'mailpoet_has_valid_premium_key' => $this->subscribersFeature->hasValidPremiumKey(),
+      'mailpoet_has_premium_support' => $this->subscribersFeature->hasPremiumSupport(),
+      'mailpoet_plugin_partial_key' => $this->servicesChecker->generatePartialApiKey(),
+      'mailpoet_subscribers_count' => $this->subscribersFeature->getSubscribersCount(),
+      'mailpoet_subscribers_limit' => $this->subscribersFeature->getSubscribersLimit(),
+      'mailpoet_subscribers_limit_reached' => $this->subscribersFeature->check(),
+    ];
+    $this->wp->wpAddInlineScript( 'mailpoet_email_editor', implode( '', array_map( function($key) use ($inline_script_data) {
+      return sprintf( "var %s=%s;", $key, json_encode( $inline_script_data[$key] ) );
+    }, array_keys( $inline_script_data ) ) ), 'before');
```

Comment:
The implementation looks good. However, we don't use a space after the opening bracket or before the closing bracket. We use a space after the keyword `function'. I tried to prepare a suggestion because I know the code style is different from what you are used to.

```suggestion
    $this->wp->wpAddInlineScript('mailpoet_email_editor', implode('', array_map(function ($key) use ($inline_script_data) {
      return sprintf("var %s=%s;", $key, json_encode($inline_script_data[$key]));
    }, array_keys($inline_script_data))), 'before');
```

Reply:
phpcs didn't format this automatically -.-

Reply:
Unfortunately, it doesn't work now, but Rosta created [a ticket](https://mailpoet.atlassian.net/browse/MAILPOET-5946) to fix that